test/auth/custom-strategy/int.spec.ts

```diff
@@ -0,0 +1,57 @@
+import mongoose from 'mongoose';
```

Comment:
This is beautiful. Love the tests! We haven't even written the contributing guide that covers this and you're already doing it!